composer.json

```diff
@@ -20,15 +20,26 @@
             "homepage": "https://github.com/friendsofsymfony/FOSHttpCache/contributors"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/ddeboer/discovery.git"
```

Comment:
TODO: remove when https://github.com/php-http/discovery/pull/9 is merged and tagged.